Bottom: 396ab9a496814b0a19290fdcfd511ca8549588a8
Top:    aea7d42cd22623bf012eb186e389cb43754cea36
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:08 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index d7b5e34..ae0b9fa 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
